x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/cluster/routing/allocation/DataTierIT.java

```diff
@@ -33,13 +33,42 @@
         return Collections.singleton(LocalStateCompositeXPackPlugin.class);
     }
 
-    public void testDefaultAllocateToHot() {
+    public void testDefaultIndexAllocateToContent() {
         startWarmOnlyNode();
         startColdOnlyNode();
         ensureGreen();
 
         client().admin().indices().prepareCreate(index).setWaitForActiveShards(0).get();
 
+        Settings idxSettings = client().admin().indices().prepareGetIndex().addIndices(index).get().getSettings().get(index);
+        assertThat(DataTierAllocationDecider.INDEX_ROUTING_INCLUDE_SETTING.get(idxSettings), equalTo(DataTier.DATA_CONTENT));
+
+        // index should be red
+        assertThat(client().admin().cluster().prepareHealth(index).get().getIndices().get(index).getStatus(),
+            equalTo(ClusterHealthStatus.RED));
+
+        logger.info("--> starting hot node");
```

Comment:
nit: this log message should be moved inside the if/else below and adapted accordinly